spring-boot-project/spring-boot-devtools/src/test/java/org/springframework/boot/devtools/livereload/ConnectionInputStreamTests.java

```diff
@@ -37,13 +37,15 @@
 	private static final byte[] NO_BYTES = {};
 
 	@Test
-	void readHeader() throws Exception {
-		String header = "";
+	public void readHeader() throws Exception {
```

Comment:
A Junit Jupiter test doesn't have to be public.

Reply:
> A Junit Jupiter test doesn't have to be public.

ok,thanks